modules/apps/commerce/commerce-product-content-api/bnd.bnd

```diff
@@ -1,6 +1,6 @@
 Bundle-Name: Liferay Commerce Product Content API
 Bundle-SymbolicName: com.liferay.commerce.product.content.api
-Bundle-Version: 11.0.1
+Bundle-Version: 12.0.0
```

Comment:
this breaking change needs to be documented with a commit description
Reference: https://liferay.atlassian.net/wiki/spaces/ENGR/pages/2489483265/Process+to+register+breaking+changes+in+commits
Example:
https://github.com/brianchandotcom/liferay-portal/pull/142322/commits/d5dde9bfd099fff1cf8f211ebd931ae1377e7fe4